server/src/termdb.gdc.js

```diff
@@ -998,8 +999,20 @@ async function cacheSampleIdMapping(ds) {
 	initGdcHolder(ds)
 
 	try {
+		// also indicates to ignore data_release_version even if present
+		const response = await ds.preInit.getStatus()
+		if (deepEqual(response.data_release_version, ds.__gdc.data_release_version) && ds.__gdc.doneCaching) {
```

Comment:
duplicate logic with caseCacheIsStale(). if you want to go this way, delete duplicate